alerting/provider/teamsworkflow/teamsworkflow.go

```diff
@@ -0,0 +1,182 @@
+package teamsworkflow
```

Comment:
```suggestion
package teamsworkflows
```
the path should also be changed.